packages/aml-checklist/package.json

```diff
@@ -27,6 +27,7 @@
   "dependencies": {
     "@reapit/elements": "^0.5.39",
     "@reapit/cognito-auth": "^2.0.19",
+    "@reapit/utils": "^0.0.1",
```

Comment:
Do we need to add this ?

Reply:
Yes, to rerun test when we change something inside `packages/utils`